src/nxdoc/nuxeo-server/administration/dlq.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Dead Letter Queue (DLQ)
```

Comment:
Work Dead Letter Queue
the concept of DLQ could be applied soon to other processing, so better to be specific and talk about the Work or WorkManager DLQ